src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs

```diff
@@ -106,7 +106,7 @@ public static ImmutableArray<T> CreateRange<T>(IEnumerable<T> items)
             var immutableArray = items as IImmutableArray;
             if (immutableArray != null)
             {
-                Array array = immutableArray.Array;
+                Array? array = immutableArray.Array;
```

Comment:
Should `ImmutableArray<T> Create<T>(params T[] items)` accept nullable array? GH won't let me comment there.

Reply:
yes it is